common/channelconfig/applicationorg.go

```diff
@@ -54,6 +60,12 @@ func NewApplicationOrgConfig(id string, orgGroup *cb.ConfigGroup, mspConfig *MSP
 		},
 	}
 
+	if !aoc.Capabilities().PrivateDataImplicitCollectionConfig() {
+		if _, ok := orgGroup.Values[PrivateDataImplicitCollectionKey]; ok {
```

Comment:
Hmmm, I tried not configuring V2_0 capability while PrivateDataImplicitCollection was configured... it got to this 'if' statement, but didn't enter it (didn't return the error as I expected).

Reply:
Weird, maybe the key was mispelled. I'll take a look.

Reply:
I think I'm doing something wrong with how the orgGroup.Values is being built since it can't even seem to find the `PrivateDataImplicitCollectionKey` in the list of values.